examples/amm/src/contract.rs

```diff
@@ -70,49 +69,19 @@ impl Contract for AmmContract {
 
         match message {
             Message::Swap {
-                owner,
+                account,
```

Comment:
We could just always assume that the chain ID is the one that sent the message, and keep this field only as `owner`?